### PR TITLE
Add transforms

### PR DESCRIPTION
**Pull request by danielballan**
This is a work in progress, will be expanded as we find more transforms that we need.

### CODE REVIEW

File: nslsii/transforms.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# DataBroker "transforms" for patching up documents
+# on their way out.
+import copy
+
+
+def fix_csx_scaler_shape(d)
```
Comment (gwbischof) [inline nslsii/transforms.py]: missing a :

**Comment by gwbischof** [comment]
What if we put the beamline name first? And no beamline name if it can apply to other beamlines?

File: nslsii/transforms.py (hunk 1 above)
Comment (danielballan) [reply nslsii/transforms.py]: D'oh!

**Comment by danielballan** [comment]
That works for me. Change pushed.

**Pull request merged**